org.kde.krita.yaml

```diff
@@ -14,7 +14,6 @@ finish-args:
   - --socket=pulseaudio
   - --device=dri
   - --filesystem=host
-  - --filesystem=xdg-config/kdeglobals:ro
```

Comment:
Do you know shy it was needed before and not anymore?

Reply:
The file contains information for KDE Plasma's system theme like Breeze and the colors. But the Krita flatpak uses it's own styling with Fusion. If we are able to switch to the KDE Runtime in the future as well it is specified there as well. https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/qt5.15lts-21.08/org.kde.Sdk.json.in#L78